recipes/topologic-core/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "topologic-core" %}
+{% set version = "8.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wassimj/Topologic/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c29953364d91f8523403f9f7c244d1e6462243027882273a52731cdc9bf1238e
+  patches:
+    - patches/do_not_use_pybind11_subdir.patch
+
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
```

Comment:
This is a Python package, why does it have a run_exports?

Reply:
Yeah, I'll remove that for both packages now.